mobile/assets/i18n/en-US.json

```diff
@@ -1,4 +1,35 @@
 {
+  "location_permission": "Location permission",
+  "location_permission_content": "Immich needs your permission to access the location permission to read the WiFi name",
+  "background_location_permission": "Background location permission",
+  "background_location_permission_content": "Immich needs your permission to ALWAYS have access to the location permission to read the WiFi name when the app is uploading assets in the background",
+  "current_server_address": "Current server address",
+  "grant_permission": "Grant permission",
+  "automatic_endpoint_switching_title": "Automatic URLs switching",
+  "automatic_endpoint_switching_subtitle": "Switch between URLs automatically when on or off designated Wi-Fi networks",
+  "local_network": "Local network",
+  "local_network_sheet_info": "When connect to the following Wi-Fi network, the app will prioritize connecting to the server at the following endpoint",
+  "external_network": "External network",
+  "external_network_sheet_info": "When not on the preferred WiFi network, the app will attempt to connect to the server at the following endpoints from top to bottom",
```

Comment:
```suggestion
  "external_network_sheet_info": "When not on the preferred WiFi network, the app will connect to the server through the first of the below URLs it can reach, starting from top to bottom",
```